docker-compose.yml

```diff
@@ -163,6 +165,40 @@ services:
     deploy:
       mode: global
 
+  # audius-cmd
+
+  audius-cmd:
+    build: dev-tools/commands
+    environment:
+      ETH_TOKEN_ADDRESS: "${ETH_TOKEN_ADDRESS}"
+      ETH_REGISTRY_ADDRESS: "${ETH_REGISTRY_ADDRESS}"
+      ETH_PROVIDER_URL: "http://eth-ganache:8545"
+      ETH_OWNER_WALLET: "${ETH_OWNER_WALLET}"
+
+      POA_REGISTRY_ADDRESS: "${POA_REGISTRY_ADDRESS}"
+      POA_PROVIDER_URL: "http://poa-ganache:8545"
+
+      SOLANA_ENDPOINT: "http://solana-test-validator:8899"
+      SOLANA_OWNER_SECRET_KEY: "${SOLANA_OWNER_SECRET_KEY}"
+      SOLANA_TOKEN_MINT_PUBLIC_KEY: "${SOLANA_TOKEN_MINT_PUBLIC_KEY}"
+      SOLANA_CLAIMABLE_TOKENS_PUBLIC_KEY: "${SOLANA_CLAIMABLE_TOKENS_PUBLIC_KEY}"
+      SOLANA_REWARD_MANAGER_PUBLIC_KEY: "${SOLANA_REWARD_MANAGER_PUBLIC_KEY}"
+      SOLANA_REWARD_MANAGER_PDA_PUBLIC_KEY: "${SOLANA_REWARD_MANAGER_PDA_PUBLIC_KEY}"
+      SOLANA_REWARD_MANAGER_TOKEN_PDA_PUBLIC_KEY: "${SOLANA_REWARD_MANAGER_TOKEN_PDA_PUBLIC_KEY}"
+      SOLANA_FEEPAYER_SECRET_KEY: "${SOLANA_FEEPAYER_SECRET_KEY}"
+
+      IDENTITY_SERVICE_URL: "http://identity-service:7000"
+      FALLBACK_CREATOR_NODE_URL: "http://creator-node:4000" # this should choose a random cn everytime this is run
+    volumes:
+      - audius-libs:/usr/src/app/node_modules/@audius/sdk
+      - ./dev-tools/commands/src:/usr/src/app/src
```

Comment:
Stomping `/usr/src/app/src` with `./dev-tools/commands/src` seems weird. It would make more sense to keep the mappings 1-to-1 and match what's being baked into the Dockerfile, but I can see how this might be a node-specific path requirement.

Reply:
They are both the same directory, it exists for hot reload functionality